packages/genesys-spark-components/src/components/beta/gux-cta-group/gux-cta-group.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, h, JSX, Prop, Element, State } from '@stencil/core';
+
+import { GuxCTAGroupAlignment } from './gux-cta-group.types';
+import { logWarn } from '@utils/error/log-error';
+
+/**
+ * As the name suggests CTA group component is used for displaying 2-3 CTAs at the end of forms or dialog-type components
+ * @prop {GuxCTAGroupAlignment} align - Defines the aligment of the buttons within the component, start or end
+ */
+@Component({
+  styleUrl: 'gux-cta-group.scss',
+  tag: 'gux-cta-group',
+  shadow: true
+})
+export class GuxCTAGroup {
+  /*
+   * Reference to the host element.
+   */
+  @Element()
+  root: HTMLElement;
+
+  /**
+   * Buttons alignment
+   */
+  @Prop()
+  align: GuxCTAGroupAlignment = 'start';
+  isDangerous: boolean = false;
+
+  @State()
+  primaryButton: Element;
+  secondaryButton: Element;
+  dismissButton: Element;
+
+  private validateSlotContent(slottedElement: Element): void {
+    if (slottedElement) {
+      const slottedTagName = slottedElement.tagName;
+      if (
+        slottedTagName.match(
+          /(BUTTON)|(GUX-BUTTON) | (GUX-ACTION-BUTTON) | (GUX-BUTTON-MULTI) | (GUX-BUTTON-SLOT)/
+        )
+      ) {
+        return;
+      }
+      if (slottedTagName.match('INPUT')) {
```

Comment:
Should it accept `input`? I think a `input` would have to be slotted into `gux-button-slot`.